### ISSUE TITLE
QA Ph4: Accessibility_Screen reader does not read "I/O is starting soon..." 

### ISSUE THREAD

**Issue by Remiqa** [GoogleChrome/ioweb2016#916]
With the NVDA screen reader enabled, mousing over the "I/O is starting soon. #io16" reads "Google I/O 2016". 

Steps: 
1. Enable the NVDA screen reader tool 
2. Go to https://io-webapp-staging.appspot.com/io2016/
3. Mouse over  "I/O is starting soon. #io16" 
4. Note issue

MS Edge Windows 10

![gio_916](https://cloud.githubusercontent.com/assets/7585604/15255747/ccf881a2-18f2-11e6-8216-9acbc1877fc3.PNG)
